.changeset/new-radios-flash.md

```diff
@@ -0,0 +1,6 @@
+---
+'@graphql-codegen/visitor-plugin-common': patch
```

Comment:
I assume that `patch` because this is kinda considered an internal API so it will not effect other users?

Reply:
Yes, internal API is also my thinking here. I'm happy to go either `minor` or `path` in this case though 🙂